lib/Files/TemplateLoader.php

```diff
@@ -59,31 +59,8 @@ public function loadTalkSidebarForFilesApp() {
 		Util::addStyle('spreed', 'autocomplete');
 		Util::addStyle('spreed', 'video');
 
-		Util::addScript('spreed', 'vendor/backbone/backbone-min');
-		Util::addScript('spreed', 'vendor/backbone.radio/build/backbone.radio.min');
-		Util::addScript('spreed', 'vendor/backbone.marionette/lib/backbone.marionette.min');
-		Util::addScript('spreed', 'vendor/handlebars/handlebars.min');
-		Util::addScript('spreed', 'vendor/jshashes/hashes.min');
-		Util::addScript('spreed', 'vendor/Caret.js/dist/jquery.caret.min');
-		Util::addScript('spreed', 'vendor/At.js/dist/js/jquery.atwho.min');
-		Util::addScript('spreed', 'models/chatmessage');
-		Util::addScript('spreed', 'models/chatmessagecollection');
-		Util::addScript('spreed', 'models/room');
-		Util::addScript('spreed', 'models/roomcollection');
-		Util::addScript('spreed', 'views/callbutton');
-		Util::addScript('spreed', 'views/chatview');
-		Util::addScript('spreed', 'views/editabletextlabel');
-		Util::addScript('spreed', 'views/emptycontentview');
-		Util::addScript('spreed', 'views/mediacontrolsview');
-		Util::addScript('spreed', 'views/richobjectstringparser');
-		Util::addScript('spreed', 'views/templates');
-		Util::addScript('spreed', 'views/virtuallist');
-		Util::addScript('spreed', 'simplewebrtc');
-		Util::addScript('spreed', 'webrtc');
-		Util::addScript('spreed', 'signaling');
-		Util::addScript('spreed', 'connection');
-		Util::addScript('spreed', 'embedded');
-		Util::addScript('spreed', 'filesplugin');
+		Util::addStyle('spreed', 'merged-embeded');
```

Comment:
`Util::addStyle('spreed', 'merged-embeded');` should replace the `Util::addStyle(...)` calls above.

Reply:
You mean git add before git commit would help? I guess so too...